examples/apps/Django_Panel_Multi/.gitignore

```diff
@@ -0,0 +1,215 @@
+
+# Created by https://www.gitignore.io/api/django,pycharm
```

Comment:
Do we need a separate .gitignore for this directory?

Reply:
No